ui/app/templates/components/secret-edit.hbs

```diff
@@ -33,75 +33,15 @@
     </ToolbarFilters>
   {{/unless}}
   <ToolbarActions>
-    {{#if (and (eq @mode "show") this.isV2 (not @model.failedServerRead))}}
```

Comment:
the ordering of toolbar-actions changed.  So now deletion comes first and version is scooted to the far right.